# --- START: seq=84 ---
while parsing a block collection
  in "<unicode string>", line 30, column 11:
              - name: MEMCACHED_CACHE_SIZE
              ^
expected <block end>, but found '?'
  in "<unicode string>", line 34, column 11:
              image: memcached
              ^
# --- END: seq=84 ---

# --- START: seq=286 ---
while parsing a block collection
  in "<unicode string>", line 30, column 11:
              - name: MEMCACHED_CACHE_SIZE
              ^
expected <block end>, but found '?'
  in "<unicode string>", line 34, column 11:
              image: memcached
              ^
# --- END: seq=286 ---

# --- START: seq=446 ---
{'max_throughput': 189.9, 'total_errors': 1639, 'total_anomaly_count': 41, 'changed_service_anomaly_count': 15, 'service_max_cpu_usage': nan, 'total_max_cpu_usage': nan, 'service_names': ['srv-rate']}
# --- END: seq=446 ---

# --- START: seq=452 ---
(400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Audit-Id': '50321ffe-381a-4722-82aa-1c7fa514eb07', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'Warning': '299 - "Kubernetes default value is insecure, Knative may default this to secure in a future release: spec.template.spec.containers[0].securityContext.allowPrivilegeEscalation, spec.template.spec.containers[0].securityContext.capabilities, spec.template.spec.containers[0].securityContext.runAsNonRoot, spec.template.spec.containers[0].securityContext.seccompProfile"', 'X-Kubernetes-Pf-Flowschema-Uid': 'b7893084-19b7-431d-9ddc-f75ccbbac163', 'X-Kubernetes-Pf-Prioritylevel-Uid': 'd5b7ccb8-eaf7-4ee2-b6f9-fdc0c0ff65cc', 'Date': 'Tue, 29 Jul 2025 14:53:41 GMT', 'Content-Length': '341'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"admission webhook \"validation.webhook.serving.knative.dev\" denied the request: validation failed: expected 110 \u003c= 100.0 \u003c= 1000: spec.template.metadata.annotations.autoscaling.knative.dev/panic-threshold-percentage","reason":"BadRequest","code":400}


# --- END: seq=452 ---

# --- START: seq=143 ---
{'max_throughput': 172.4, 'total_errors': 825, 'total_anomaly_count': 25, 'changed_service_anomaly_count': 5, 'service_max_cpu_usage': 0.9533000562636073, 'total_max_cpu_usage': nan, 'service_names': ['srv-rate']}
# --- END: seq=143 ---

# --- START: seq=457 ---
{'max_throughput': 203.2, 'total_errors': 3211, 'total_anomaly_count': 32, 'changed_service_anomaly_count': 12, 'service_max_cpu_usage': nan, 'total_max_cpu_usage': nan, 'service_names': ['srv-rate']}
# --- END: seq=457 ---

# --- START: seq=703 ---
{'max_throughput': 197.4, 'total_errors': 1661, 'total_anomaly_count': 28, 'changed_service_anomaly_count': 7, 'service_max_cpu_usage': nan, 'total_max_cpu_usage': nan, 'service_names': ['srv-rate']}
# --- END: seq=703 ---

# --- START: seq=136 ---
(400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Audit-Id': '913815cd-cbd1-4020-b0e8-c76e1003a8ce', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'Warning': '299 - "Kubernetes default value is insecure, Knative may default this to secure in a future release: spec.template.spec.containers[0].securityContext.allowPrivilegeEscalation, spec.template.spec.containers[0].securityContext.capabilities, spec.template.spec.containers[0].securityContext.runAsNonRoot, spec.template.spec.containers[0].securityContext.seccompProfile"', 'X-Kubernetes-Pf-Flowschema-Uid': 'b7893084-19b7-431d-9ddc-f75ccbbac163', 'X-Kubernetes-Pf-Prioritylevel-Uid': 'd5b7ccb8-eaf7-4ee2-b6f9-fdc0c0ff65cc', 'Date': 'Tue, 29 Jul 2025 15:35:19 GMT', 'Content-Length': '341'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"admission webhook \"validation.webhook.serving.knative.dev\" denied the request: validation failed: expected 110 \u003c= 100.0 \u003c= 1000: spec.template.metadata.annotations.autoscaling.knative.dev/panic-threshold-percentage","reason":"BadRequest","code":400}


# --- END: seq=136 ---

# --- START: seq=356 ---
(400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Audit-Id': '95e7d9c6-8fea-469f-8f73-9acc8779ad39', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'Warning': '299 - "Kubernetes default value is insecure, Knative may default this to secure in a future release: spec.template.spec.containers[0].securityContext.allowPrivilegeEscalation, spec.template.spec.containers[0].securityContext.capabilities, spec.template.spec.containers[0].securityContext.runAsNonRoot, spec.template.spec.containers[0].securityContext.seccompProfile"', 'X-Kubernetes-Pf-Flowschema-Uid': 'b7893084-19b7-431d-9ddc-f75ccbbac163', 'X-Kubernetes-Pf-Prioritylevel-Uid': 'd5b7ccb8-eaf7-4ee2-b6f9-fdc0c0ff65cc', 'Date': 'Tue, 29 Jul 2025 15:36:19 GMT', 'Content-Length': '341'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"admission webhook \"validation.webhook.serving.knative.dev\" denied the request: validation failed: expected 110 \u003c= 100.0 \u003c= 1000: spec.template.metadata.annotations.autoscaling.knative.dev/panic-threshold-percentage","reason":"BadRequest","code":400}


# --- END: seq=356 ---

# --- START: seq=103 ---
(400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'd0e20016-a36f-47b8-8d1d-831066567f48', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'Warning': '299 - "Kubernetes default value is insecure, Knative may default this to secure in a future release: spec.template.spec.containers[0].securityContext.allowPrivilegeEscalation, spec.template.spec.containers[0].securityContext.capabilities, spec.template.spec.containers[0].securityContext.runAsNonRoot, spec.template.spec.containers[0].securityContext.seccompProfile"', 'X-Kubernetes-Pf-Flowschema-Uid': 'b7893084-19b7-431d-9ddc-f75ccbbac163', 'X-Kubernetes-Pf-Prioritylevel-Uid': 'd5b7ccb8-eaf7-4ee2-b6f9-fdc0c0ff65cc', 'Date': 'Tue, 29 Jul 2025 15:37:13 GMT', 'Content-Length': '341'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"admission webhook \"validation.webhook.serving.knative.dev\" denied the request: validation failed: expected 110 \u003c= 100.0 \u003c= 1000: spec.template.metadata.annotations.autoscaling.knative.dev/panic-threshold-percentage","reason":"BadRequest","code":400}


# --- END: seq=103 ---

# --- START: seq=330 ---
(400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'c7d93625-95cc-4910-b56d-9bfb7e3208c7', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'Warning': '299 - "Kubernetes default value is insecure, Knative may default this to secure in a future release: spec.template.spec.containers[0].securityContext.allowPrivilegeEscalation, spec.template.spec.containers[0].securityContext.capabilities, spec.template.spec.containers[0].securityContext.runAsNonRoot, spec.template.spec.containers[0].securityContext.seccompProfile"', 'X-Kubernetes-Pf-Flowschema-Uid': 'b7893084-19b7-431d-9ddc-f75ccbbac163', 'X-Kubernetes-Pf-Prioritylevel-Uid': 'd5b7ccb8-eaf7-4ee2-b6f9-fdc0c0ff65cc', 'Date': 'Tue, 29 Jul 2025 15:38:43 GMT', 'Content-Length': '340'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"admission webhook \"validation.webhook.serving.knative.dev\" denied the request: validation failed: expected 110 \u003c= 50.0 \u003c= 1000: spec.template.metadata.annotations.autoscaling.knative.dev/panic-threshold-percentage","reason":"BadRequest","code":400}


# --- END: seq=330 ---

# --- START: seq=484 ---
{'max_throughput': 181.3, 'total_errors': 628, 'total_anomaly_count': 29, 'changed_service_anomaly_count': 9, 'service_max_cpu_usage': nan, 'total_max_cpu_usage': nan, 'service_names': ['srv-rate', 'config-autoscaler']}
# --- END: seq=484 ---

# --- START: seq=480 ---
(400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'cbddd9da-6309-4d4a-9119-ec97298fc626', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'Warning': '299 - "Kubernetes default value is insecure, Knative may default this to secure in a future release: spec.template.spec.containers[0].securityContext.allowPrivilegeEscalation, spec.template.spec.containers[0].securityContext.capabilities, spec.template.spec.containers[0].securityContext.runAsNonRoot, spec.template.spec.containers[0].securityContext.seccompProfile"', 'X-Kubernetes-Pf-Flowschema-Uid': 'b7893084-19b7-431d-9ddc-f75ccbbac163', 'X-Kubernetes-Pf-Prioritylevel-Uid': 'd5b7ccb8-eaf7-4ee2-b6f9-fdc0c0ff65cc', 'Date': 'Tue, 29 Jul 2025 15:54:40 GMT', 'Content-Length': '341'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"admission webhook \"validation.webhook.serving.knative.dev\" denied the request: validation failed: expected 110 \u003c= 100.0 \u003c= 1000: spec.template.metadata.annotations.autoscaling.knative.dev/panic-threshold-percentage","reason":"BadRequest","code":400}


# --- END: seq=480 ---

# --- START: seq=614 ---
(400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'bcd0ce43-2f9f-427d-ab36-cca16e6e7a0c', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'Warning': '299 - "Kubernetes default value is insecure, Knative may default this to secure in a future release: spec.template.spec.containers[0].securityContext.allowPrivilegeEscalation, spec.template.spec.containers[0].securityContext.capabilities, spec.template.spec.containers[0].securityContext.runAsNonRoot, spec.template.spec.containers[0].securityContext.seccompProfile"', 'X-Kubernetes-Pf-Flowschema-Uid': 'b7893084-19b7-431d-9ddc-f75ccbbac163', 'X-Kubernetes-Pf-Prioritylevel-Uid': 'd5b7ccb8-eaf7-4ee2-b6f9-fdc0c0ff65cc', 'Date': 'Tue, 29 Jul 2025 15:55:43 GMT', 'Content-Length': '341'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"admission webhook \"validation.webhook.serving.knative.dev\" denied the request: validation failed: expected 110 \u003c= 100.0 \u003c= 1000: spec.template.metadata.annotations.autoscaling.knative.dev/panic-threshold-percentage","reason":"BadRequest","code":400}


# --- END: seq=614 ---

# --- START: seq=646 ---
{'max_throughput': 202.6, 'total_errors': 4962, 'total_anomaly_count': 22, 'changed_service_anomaly_count': 7, 'service_max_cpu_usage': 1.17412902368878, 'total_max_cpu_usage': nan, 'service_names': ['srv-reservation', 'config-autoscaler']}
# --- END: seq=646 ---

# --- START: seq=591 ---
(400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Audit-Id': '6f2964e1-7707-47de-b71f-3e73acd711d5', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'Warning': '299 - "Kubernetes default value is insecure, Knative may default this to secure in a future release: spec.template.spec.containers[0].securityContext.allowPrivilegeEscalation, spec.template.spec.containers[0].securityContext.capabilities, spec.template.spec.containers[0].securityContext.runAsNonRoot, spec.template.spec.containers[0].securityContext.seccompProfile"', 'X-Kubernetes-Pf-Flowschema-Uid': 'b7893084-19b7-431d-9ddc-f75ccbbac163', 'X-Kubernetes-Pf-Prioritylevel-Uid': 'd5b7ccb8-eaf7-4ee2-b6f9-fdc0c0ff65cc', 'Date': 'Tue, 29 Jul 2025 16:11:11 GMT', 'Content-Length': '341'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"admission webhook \"validation.webhook.serving.knative.dev\" denied the request: validation failed: expected 110 \u003c= 100.0 \u003c= 1000: spec.template.metadata.annotations.autoscaling.knative.dev/panic-threshold-percentage","reason":"BadRequest","code":400}


# --- END: seq=591 ---

# --- START: seq=428 ---
{'max_throughput': 206.7, 'total_errors': 5771, 'total_anomaly_count': 29, 'changed_service_anomaly_count': 7, 'service_max_cpu_usage': 1.015082125734075, 'total_max_cpu_usage': nan, 'service_names': ['srv-rate']}
# --- END: seq=428 ---

# --- START: seq=768 ---
{'max_throughput': 191.2, 'total_errors': 4166, 'total_anomaly_count': 32, 'changed_service_anomaly_count': 9, 'service_max_cpu_usage': 0.9566016352609916, 'total_max_cpu_usage': nan, 'service_names': ['srv-rate']}
# --- END: seq=768 ---

# --- START: seq=154 ---
(400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Audit-Id': '4a453fc9-6c47-45c8-81ed-91300ca190d8', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'Warning': '299 - "Kubernetes default value is insecure, Knative may default this to secure in a future release: spec.template.spec.containers[0].securityContext.allowPrivilegeEscalation, spec.template.spec.containers[0].securityContext.capabilities, spec.template.spec.containers[0].securityContext.runAsNonRoot, spec.template.spec.containers[0].securityContext.seccompProfile"', 'X-Kubernetes-Pf-Flowschema-Uid': 'b7893084-19b7-431d-9ddc-f75ccbbac163', 'X-Kubernetes-Pf-Prioritylevel-Uid': 'd5b7ccb8-eaf7-4ee2-b6f9-fdc0c0ff65cc', 'Date': 'Tue, 29 Jul 2025 16:39:32 GMT', 'Content-Length': '342'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"admission webhook \"validation.webhook.serving.knative.dev\" denied the request: validation failed: expected 110 \u003c= 2000.0 \u003c= 1000: spec.template.metadata.annotations.autoscaling.knative.dev/panic-threshold-percentage","reason":"BadRequest","code":400}


# --- END: seq=154 ---

# --- START: seq=333 ---
{'max_throughput': 192.7, 'total_errors': 6721, 'total_anomaly_count': 21, 'changed_service_anomaly_count': 0, 'service_max_cpu_usage': -1, 'total_max_cpu_usage': nan, 'service_names': ['memcached-rate']}
# --- END: seq=333 ---

# --- START: seq=551 ---
(400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'ef53a375-b3ac-467a-93f5-80f2f76c3976', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'b7893084-19b7-431d-9ddc-f75ccbbac163', 'X-Kubernetes-Pf-Prioritylevel-Uid': 'd5b7ccb8-eaf7-4ee2-b6f9-fdc0c0ff65cc', 'Date': 'Tue, 29 Jul 2025 16:54:30 GMT', 'Content-Length': '260'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"admission webhook \"config.webhook.serving.knative.dev\" denied the request: validation failed: max-scale-down-rate = 1, must be greater than 1.0","reason":"BadRequest","code":400}


# --- END: seq=551 ---

# --- START: seq=774 ---
{'max_throughput': 238.6, 'total_errors': 6743, 'total_anomaly_count': 29, 'changed_service_anomaly_count': 0, 'service_max_cpu_usage': -1, 'total_max_cpu_usage': nan, 'service_names': ['config-autoscaler']}
# --- END: seq=774 ---

# --- START: seq=349 ---
while parsing a block collection
  in "<unicode string>", line 30, column 11:
              - name: MEMCACHED_CACHE_SIZE
              ^
expected <block end>, but found '?'
  in "<unicode string>", line 36, column 11:
              image: memcached
              ^
# --- END: seq=349 ---

# --- START: seq=169 ---
while parsing a block collection
  in "<unicode string>", line 46, column 11:
              - name: MEMCACHED_CACHE_SIZE
              ^
expected <block end>, but found '?'
  in "<unicode string>", line 50, column 11:
              image: memcached
              ^
# --- END: seq=169 ---

# --- START: seq=387 ---
while parsing a block collection
  in "<unicode string>", line 30, column 11:
              - name: MEMCACHED_CACHE_SIZE
              ^
expected <block end>, but found '?'
  in "<unicode string>", line 34, column 11:
              image: memcached
              ^
# --- END: seq=387 ---

# --- START: seq=442 ---
while parsing a block collection
  in "<unicode string>", line 30, column 11:
              - name: MEMCACHED_CACHE_SIZE
              ^
expected <block end>, but found '?'
  in "<unicode string>", line 34, column 11:
              image: memcached
              ^
# --- END: seq=442 ---

# --- START: seq=373 ---
while parsing a block collection
  in "<unicode string>", line 30, column 11:
              - name: MEMCACHED_CACHE_SIZE
              ^
expected <block end>, but found '?'
  in "<unicode string>", line 34, column 11:
              image: memcached
              ^
# --- END: seq=373 ---

# --- START: seq=528 ---
while parsing a block collection
  in "<unicode string>", line 30, column 11:
              - name: MEMCACHED_CACHE_SIZE
              ^
expected <block end>, but found '?'
  in "<unicode string>", line 34, column 11:
              image: memcached
              ^
# --- END: seq=528 ---

# --- START: seq=166 ---
while parsing a block collection
  in "<unicode string>", line 30, column 11:
              - name: MEMCACHED_CACHE_SIZE
              ^
expected <block end>, but found '?'
  in "<unicode string>", line 34, column 11:
              image: memcached
              ^
# --- END: seq=166 ---

# --- START: seq=764 ---
while parsing a block collection
  in "<unicode string>", line 30, column 11:
              - name: MEMCACHED_CACHE_SIZE
              ^
expected <block end>, but found '?'
  in "<unicode string>", line 34, column 11:
              image: memcached
              ^
# --- END: seq=764 ---

# --- START: seq=767 ---
{'max_throughput': 222.6, 'total_errors': 3167, 'total_anomaly_count': 34, 'changed_service_anomaly_count': 4, 'service_max_cpu_usage': 0.1046624664774928, 'total_max_cpu_usage': nan, 'service_names': ['srv-profile']}
# --- END: seq=767 ---

